staging/src/k8s.io/legacy-cloud-providers/azure/azure_backoff.go

```diff
@@ -667,7 +667,7 @@ func (az *Cloud) CreateOrUpdateVmssWithRetry(resourceGroupName string, VMScaleSe
 		// When vmss is being deleted, CreateOrUpdate API would report "the vmss is being deleted" error.
 		// Since it is being deleted, we shouldn't send more CreateOrUpdate requests for it.
 		klog.V(3).Infof("CreateOrUpdateVmssWithRetry: verify the status of the vmss being created or updated")
-		vmss, err := az.VirtualMachineScaleSetsClient.Get(ctx, resourceGroupName, VMScaleSetName)
+		vmss, _ := az.VirtualMachineScaleSetsClient.Get(ctx, resourceGroupName, VMScaleSetName)
```

Comment:
You should handle this error instead of ignoring it.

Reply:
let's just warning about the err, since the vmss handling is just below this line, it will do all the real job.